src/Services/Air/validators/reservation-locator.js

```diff
@@ -0,0 +1,8 @@
+import { AirValidationError } from '../AirErrors';
+
+export default (params) => {
+  if (params.uapi_reservation_locator === undefined
```

Comment:
I'm not sure about format. Is it base64-encoded? You may check it with regexp then. If not, that's ok.